rules/Transform/Rector/ClassMethod/WrapReturnRector.php

```diff
@@ -107,22 +108,21 @@ public function configure(array $configuration): void
         $this->typeMethodWraps = $configuration;
     }
 
-    private function wrap(ClassMethod $classMethod, bool $isArrayWrap): ?ClassMethod
+    private function wrap(ClassMethod $classMethod): bool
```

Comment:
Instead of returning the ClassMethod (which was not used), we return a boolean indicating if the value has changed